spec/features/admin/move_advisers_spec.rb

```diff
@@ -51,13 +51,10 @@ def and_i_want_to_move_adviser(adviser)
 
   def and_i_want_to_move_to_firm(firm)
     expect(choose_firm_page).to be_displayed
-    expect(choose_firm_page.hidden.advisers[0].value).to eq(adviser.id.to_s)
```

Comment:
I've not looked at the full scenario of what this should be doing, but rather than just deleting these tests, should we be looking to cover this in another way? 
I.e. checking that we cannot see certain advisers on the page?

Reply:
If there were much value in the checks, I'd tend to agree.

But this is an end-to-end spec about moving and adviser from one firm to another, which will fail if those hidden inputs are not present, the checks are redundant.

On balance I can't see it being worth the overhead of using something other than Capybara for checking the html in this one case. It will add to the overhead of understanding the code.

Reply:
@willhall88 If you did want to retain these checks, what do you reckon we should use? 

(I'm thinking regex on the html.)

Reply:
ok, if you've checked and are sure the tests will still fail and those expectations are redundant then I am happy that they can go.

Reply:
✌️ Yes, totally fails if we mess with the production code so that the hidden advisers and firms aren't present.